src/game/game.gd

```diff
@@ -0,0 +1,20 @@
+extends Node2D
+
+onready var mapNode: Node2D = $Map
+onready var characterNode: Node2D = $Characters
+onready var rng = RandomNumberGenerator.new()
+
+var characterTemplate: Resource = ResourceLoader.load("res://game/character/character.tscn")
+
+func loadMap(mapPath: String) -> void:
+	var mapToLoad: Node = ResourceLoader.load(mapPath).instance()
+	add_child_below_node(mapNode, mapToLoad)
+
+func addCharacter(networkID) -> void:
+	var newCharacterResource: CharacterResource = Characters.createCharacter(networkID)
+	var newCharacter: KinematicBody2D = newCharacterResource.getCharacterNode()
+	var position: Vector2
+	position.x = rng.randi_range(100, 500)
+	position.y = rng.randi_range(100, 500)
```

Comment:
This looks fine on the surface but I think this is actually changing the position of the game node, not editing the position variable you declare. I only realized this because there was an editor warning saying the position variable is never used lol

Probably should change the name of the `position` variable to something like `newPosition` or `characterPosition`